src/Integration/LocalServices/ICredentialStoreService.cs

```diff
@@ -22,7 +22,7 @@
 
 namespace SonarLint.VisualStudio.Integration
 {
-    public interface ICredentialStoreService : ILocalService
```

Comment:
I considered dropping the `Service` suffix from the interface name since it is no longer a local service, but that would have created a lot of renames. I might do it later in a separate PR.